openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml

```diff
@@ -0,0 +1,15 @@
+description: Full TV licence for a colour TV, before any discounts are applied.
+values:
+  2017-04-01: 147.00
+  2018-04-01: 150.50
+  2019-04-01: 154.50
+  2020-04-01: 157.50
+  2021-04-01: 159.00
```

Comment:
Should we uprate?

Reply:
I believe the government has frozen the licence fee recently

Reply:
https://www.gov.uk/government/news/tv-licence-fee-frozen-for-two-years